index.html

```diff
@@ -3177,6 +3177,273 @@ <h3>Definition of Roles</h3>
+                <td class="role-inherited"> </td>
+              </tr>
+              <tr>
+                <th class="role-namefrom-head" scope="row">Name From:</th>
+                <td class="role-namefrom">author</td>
+              </tr>
+              <tr>
+                <th class="role-namerequired-head" scope="row">Accessible Name Required:</th>
+                <td class="role-namerequired">False</td>
+              </tr>
+              <tr>
+                <th class="role-namerequired-inherited-head" scope="row">Inherits Name Required:</th>
+                <td class="role-namerequired-inherited"> </td>
+              </tr>
+              <tr>
+                <th class="role-childpresentational-head" scope="row">Children Presentational:</th>
+                <td class="role-childpresentational"> </td>
+              </tr>
+              <tr>
+                <th class="role-presentational-inherited-head" scope="row">Inherits Presentational:</th>
+                <td class="role-presentational-inherited"> </td>
+              </tr>
+              <tr>
+                <th class="implicit-values-head">Implicit Value for Role:</th>
+                <td class="implicit-values"> </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <div class="role">
+          <rdef>doc-pagenum</rdef>
+          <div class="role-description">
+            <p>A visible indicator of the page number.</p>
+            <p>The doc-pagenum role MUST be used on visible page numbers present in page headers and page footers. See <rref>doc-pageheader</rref> or <rref>doc-pagefooter</rref> for examples.</p>
+            <p>The current page number SHOULD also be provided as a name on page break elements. See the examples provided in <rref>doc-pagebreak</rref> for specifics.</p>
+          </div>
+          <table class="role-features">
+            <caption>Characteristics:</caption>
+            <thead>
+              <tr>
+                <th scope="col">Characteristic</th>
+                <th scope="col">Value</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th class="role-abstract-head" scope="row">Is Abstract:</th>
+                <td class="role-abstract"> </td>
+              </tr>
+              <tr>
+                <th class="role-parent-head" scope="row">Superclass Role:</th>
+                <td class="role-parent"><rref>section</rref></td>
+              </tr>
+              <tr>
+                <th class="role-children-head" scope="row">Subclass Roles:</th>
+                <td class="role-children"> </td>
+              </tr>
+              <tr>
+                <th class="role-base-head" scope="row">Base Concept:</th>
+                <td class="role-base"> </td>
+              </tr>
+              <tr>
+                <th class="role-related-head" scope="row">Related Concepts:</th>
+                <td class="role-related">EPUB <a
```

Comment:
No equivalent for running page number in the SSV.

Reply:
Fixed